build.gradle

```diff
@@ -257,6 +257,16 @@ subprojects {
     }
     
     apply from: deps.scripts.pmd
+
+    compileJava {
+        options.encoding = 'UTF-8'
+        options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
+    }
+
+    compileTestJava {
+        options.encoding = 'UTF-8'
+        options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
```

Comment:
Please perform this configuration in `config/gradle/javac-args.gradle`. The part with the encoding is already there.